yt_dlp/extractor/fathom.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+import time
+import json
+
+from .common import InfoExtractor
+from ..utils import (
+    get_element_html_by_id,
+    extract_attributes,
+)
+
+
+class FathomIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?fathom\.video/share/(?P<id>[^/?#&]+?)'
+    _TESTS = [{
```

Comment:
after applying suggestions, please run the tests and update as necessary. thanks!